mlrun/api/utils/clients/iguazio.py

```diff
@@ -505,18 +503,50 @@ def _generate_auth_info_from_session_verification_response(
         if x_unix_uid and x_unix_uid.lower() != "unknown":
             user_unix_id = int(x_unix_uid)
 
+        user_id, group_ids = Client._resolve_user_and_group_ids(response)
```

Comment:
I don't know what else will arrive that we might want to resolve from the body in the future so I think it better to give it a name that isn't coupled for `user` and `group_ids`, wdyt? 
```suggestion
        user_id, group_ids = self._resolve_params_from_response_body(response)
```